modules/olm-deleting-operators-from-a-cluster-using-cli.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operator-from-a-cluster-using-cli-{context}']
+= Deleting operators from a cluster using the CLI
+
+Instead of using the {product-title} web console, you can delete an Operator
+from your cluster by using the CLI.
+You do this by deleting the Subscription and ClusterServiceVersion
+from the `targetNamespace`, then editing the CatalogSourceConfig to remove
+the Operator's package name.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin`
+permissions.
+
+- Install the *oc* command on your local system.
+
+.Procedure
+
+In this example, there are two operators (Jaeger and Descheduler) installed in the
+`openshift-operators` namespace. The goal is to remove Jaeger without removing Descheduler.
+
+. Delete the Operator's subscription (for example, jaeger):
++
+----
+$ oc delete subscription jaeger -n openshift-operators
+subscription.operators.coreos.com "jaeger" deleted
+----
+
+. Check the current version of the subscribed Operator (for example, jaeger):
++
+----
+$ oc get subscription jaeger -n openshift-operators -o yaml | grep currentCSV
+  currentCSV: jaeger-operator.v1.8.2
+----
+
+. Delete the CSV for the Operator (jaeger) in the `targetNamespace` (`openshift-operators`)
++
+----
+$ oc delete clusterserviceversion jaeger-operator.v1.8.2 -n openshift-operators
+clusterserviceversion.operators.coreos.com "jaeger-operator.v1.8.2" deleted
+----
+
+.  Display the contents of the `CatalogSourceConfig` resource and review the list
+of packages in the `spec` section:
++
+----
+$ oc get catalogsourceconfig -n openshift-marketplace \
+    installed-community-openshift-operators -o yaml
+----
++
+For example, the spec section might appear as follows:
++
+.Example of CatalogSourceConfig
+[source,yaml]
+----
+spec:
+  csDisplayName: Community Operators
+  csPublisher: Community
+  packages: jaeger,descheduler
+  targetNamespace: openshift-operators
+----
+. Remove the Operator from the CatalogSourceConfig in one of two ways:
+
+** If there is only one Operator in the CatalogSourceConfig, you can remove it
```

Comment:
Can the multiple operators case be listed first? I think it's the more general case, whereas a single operator is a special case

Reply:
Okay. Done.